manifests/params.pp

```diff
@@ -55,40 +51,53 @@
   $whisper_workers                    = 1
   # Limits the number of whisper update_many() calls per second. 0 - no limit
   $whisper_max_updates_per_second     = 0
+  $whisper_max_creates_per_second     = 500
   $whisper_enabled                    = true
 
   # Limit of in-memory stored points (not metrics)
   $cache_max_size                     = 1000000
   # Capacity of queue between receivers and cache
-  $cache_input_buffer                 = 51200
+  # Strategy to persist metrics. Values: "max","sorted","noop"
+  #   "max" - write metrics with most unwritten datapoints first
+  #   "sorted" - sort by timestamp of first unwritten datapoint.
+  #   "noop" - pick metrics to write in unspecified order,
+  #            requires least CPU and improves cache responsiveness
+  $cache_write_strategy               = 'max'
 
-  $udp_listen                         = '0.0.0.0:2003'
+  $udp_listen                         = ':2003'
   # Enable optional logging of incomplete messages (chunked by MTU)
   $udp_log_incomplete                 = false
   $udp_enabled                        = true
+  # Optional internal queue between receiver and cache
+  $udp_buffer_size                    = 0
```

Comment:
What does a buffer of 0 indicate?

Reply:
I think 0 indicates that no buffer is configured